.changeset/early-pugs-refuse.md

```diff
@@ -0,0 +1,5 @@
+---
+"ledger-live-desktop": patch
```

Comment:
Now we need to use `minor` and not `patch` anymore

Reply:
done ☑️